doc/source/toolchain.rst

```diff
@@ -27,6 +27,7 @@ section for an example.)
 - First and foremost, SciPy is a Python project, hence it requires a Python environment.
 - BLAS and LAPACK numerical libraries need to be installed.
 - Compilers for C, C++, Cython, and Fortran code are needed.
+- Since version 1.7.0, ``Pythran`` is an optional dependency (which can be enabled via ``export SCIPY_USE_PYTHRAN=1``).
```

Comment:
On `master` it is now opt-out instead of opt-in for testing purposes. Not sure if we know if that will be "final" status for `1.7.0` just yet.